sdk/loadtestservice/Azure.Developer.LoadTesting/src/LoadTestAdministrationClient.cs

```diff
@@ -0,0 +1,344 @@
+        /// Response Body:
+        /// Schema for <c>FileUrl</c>:
+        /// <code>{
+        ///   url: string, # Optional. File URL.
+        ///   fileId: string, # Optional. File unique identifier.
+        ///   filename: string, # Optional. Name of the file.
+        ///   fileType: &quot;0&quot; | &quot;1&quot; | &quot;2&quot;, # Optional. Integer representation of the file type (0 = JMX_FILE, 1 = USER_PROPERTIES, 2 = ADDITIONAL_ARTIFACTS)
+        ///   expireTime: string (ISO 8601 Format), # Optional. Expiry time of the file
+        ///   validationStatus: string, # Optional. Validation status of the file
+        /// }
+        /// </code>
+        /// </remarks>
+        public virtual async Task<Response> UploadTestFileAsync(string testId, string fileId, string fileName, RequestContent content, int? fileType = null, RequestContext context = null)
+        {
+            Argument.AssertNotNullOrEmpty(testId, nameof(testId));
+            Argument.AssertNotNullOrEmpty(fileId, nameof(fileId));
+            Argument.AssertNotNull(content, nameof(content));
+
+            using var scope = ClientDiagnostics.CreateScope("LoadTestAdministrationClient.UploadTestFile");
+            scope.Start();
+            try
+            {
+                //Random random = new Random();
+
+                //const string chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890";
+                //string randomBoundry = new string(Enumerable.Repeat(chars, 10).Select(s => s[random.Next(s.Length)]).ToArray());
+
+                string randomBoundry = "hybboH2feH";
+                string boundary = "----WebkitFormBoundary" + randomBoundry;
+
+                String newline = "" + (char)0x0D + (char)0x0A;
+                Stream body = new MemoryStream();
+                StreamWriter writer = new StreamWriter(body);
+                writer.Write("--" + boundary + newline);
+                writer.Write("Content-Disposition: form-data; name=\"file\"; filename=\"" + fileName + "\"" + newline);
+                writer.Write("Content-Type: application/octet-stream" + newline + newline);
+                writer.Flush();
+
+                writer.Write(content);
+                //content.WriteTo(body, new System.Threading.CancellationToken());
+
+                writer.Write(newline + newline + "--" + boundary + "--");
+                writer.Flush();
+                body.Position = 0;
+                Console.WriteLine(body);
+
+                using HttpMessage message = CreateUploadTestFileRequest(testId, fileId, RequestContent.Create(body), fileType, context);
+                message.Request.Headers.SetValue("Content-Type", "multipart/form-data;boundary=" + boundary);
+                return await _pipeline.ProcessMessageAsync(message, context).ConfigureAwait(false);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
```

Comment:
Can we use [MultiPartContent](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Shared/Multipart/MultipartContent.cs) here? [Here](https://github.com/Azure/azure-sdk-for-net/blob/e29cd16283b0213e99b41376cf08542d03d6640a/sdk/tables/Azure.Data.Tables/src/TableRestClient.cs#L55) is an example of its use

Reply:
Talked to @christothes , resolving